python/pfs/drp/stella/readLineList.py

```diff
@@ -90,6 +90,8 @@ def filterByIntensity(self, lines):
         filtered : `pfs.drp.stella.ReferenceLineSet`
             Filtered list of reference lines.
         """
+        if self.config.minIntensity <= 0:
```

Comment:
It might be useful to mention in the docString and/or in the description for that parameter, that no filtering is done if the value of that parameter is <= 0.